tox.ini

```diff
@@ -12,6 +12,7 @@ passenv = SSH_AUTH_SOCK PAASTA_ENV DOCKER_HOST CI
 setenv =
     TZ = UTC
 deps =
+    --only-binary=grpcio
```

Comment:
i wonder if there's a way for us to set this in the requirements file so that we only need to specify this once - i was going to suggest adding a comment as to why this is here would be nice (i.e., we have wheels readily available internally but the same is not true in public pypi), but we'd need to copy-paste that quite a bit :p

Reply:
oh nice, it is possible: https://pip.pypa.io/en/stable/reference/requirements-file-format/#global-options

if it doesn't break requirements-tools (i.e., check-requirements) to do this, that'd make this diff smaller (we wouldn't have to add this --only-binary option to all the toxenvs nor to the dockerfiles!) and it would let us add a comment explaining why we're singling out grpcio

Reply:
(that said, also fine as a followup)